_topics/en/simplicity.md

```diff
@@ -29,21 +46,25 @@ primary_sources:
 ## true" to bold entry
 optech_mentions:
   - title: OP_CHECKTEMPLATEVERIFY
-    url: /en/newsletters/2019/12/04/#insert_hashtag
+    url: /en/newsletters/2019/12/04/#op-checktemplateverify-ctv
     date: 2019-12-04
 
   - title: Question about Simplicity and static analysis
-    url: /en/newsletters/2020/04/29/#insert_hashtag
+    url: /en/newsletters/2020/04/29/#how-is-simplicity-better-suited-for-static-analysis-compared-to-script
     date: 2020-04-29
 
   - title: Simplicity - Next Generation Smart Contracting
```

Comment:
Suggest mentioning that this was a webinar or that it's a transcript summary.  E.g.: "Transcript: next generation smart contracting with Simplicity"